mailer/message.go

```diff
@@ -0,0 +1,60 @@
+package mailer
+
+import (
+	"io"
+
+	"bytes"
+
+	"github.com/gobuffalo/buffalo/render"
+)
+
+//Message represents an Email message
+type Message struct {
+	From    string
+	To      []string
+	CC      []string
+	Bcc     []string
+	Subject string
+
+	Body        []byte
+	ContentType string
+
+	Attachments []Attachment
+}
+
+//Attachment are files added into a email message.
+type Attachment struct {
+	Name        string
+	Reader      io.Reader
+	ContentType string
+}
+
+// AddBody the message by receiving a renderer and rendering data.
+func (m *Message) AddBody(r render.Renderer, data render.Data, contentType string) error {
```

Comment:
`render.Renderer` already has a `ContentType()` method as part of the interface, do we need the param `contentType` as well?